src/sigstore.ts

```diff
@@ -44,32 +53,36 @@ type IdentityProviderOptions = Pick<
 export async function sign(
   payload: Buffer,
   options: SignOptions = {}
-): Promise<Bundle> {
+): Promise<MessageSignatureBundle> {
   const fulcio = new Fulcio({ baseURL: options.fulcioBaseURL });
   const rekor = new Rekor({ baseURL: options.rekorBaseURL });
   const idps = configureIdentityProviders(options);
-
-  return new Signer({
+  const signer = new Signer({
     fulcio,
     rekor,
     identityProviders: idps,
-  }).signBlob(payload);
+  });
+
+  const bundle = await signer.signBlob(payload);
+  return BundleSerializer.toJSON(bundle) as MessageSignatureBundle;
```

Comment:
Internal Bundle is translated to it's serialized from immediately before being returned.